modules/dnn/inference/Makefile.am

```diff
@@ -2,24 +2,22 @@ lib_LTLIBRARIES = libxcam_dnn.la
 
 XCAM_DNN_CXXFLAGS = \
     $(XCAM_CXXFLAGS)        \
+    $(LIBOPENVINO_CFLAGS)   \
     -I$(top_srcdir)/xcore   \
     -I$(top_srcdir)/modules \
-    -I$(OPENVINO_IE_INC_PATH)/ngraph/include \
-    -I$(OPENVINO_IE_INC_PATH)/inference_engine/include \
-    -I$(OPENVINO_IE_INC_PATH)/inference_engine/samples/cpp/common/format_reader \
+    -I$(OPENVINO_EXTRAS_PATH)/samples/cpp/common/format_reader/include \
     $(NULL)
 
 XCAM_DNN_LIBS = \
     $(top_builddir)/xcore/libxcam_core.la \
-    -L$(OPENVINO_IE_LIBS_PATH) \
-    -linference_engine_transformations \
-    -linference_engine \
-    -lclDNNPlugin \
-    -lMKLDNNPlugin \
-    -lGNAPlugin \
-    -lmyriadPlugin \
+    -L$(OPENVINO_EXTRAS_PATH)/bin/intel64/Release \
+    -lopenvino \
```

Comment:
try to replace these -lopenvinoxxx by $(LIBOPENVINO_LIBS) generated by pkgconfig.